### PR TITLE
API docs mini tweaks

### PR DESCRIPTION
**Pull request by julien-c**
very minor tweaks, the main one was that i didn't really like `Search API` (confusing with the Hub full text search)

### CODE REVIEW

File: docs/hub/api.md [hunk 1]
```diff
@@ -139,11 +138,6 @@ Get all information for a specific model.
 
 This is equivalent to `huggingface_hub.space_info(repo_id, revision)`.
 
-### GET /api/metrics
```
[the remark this answers is not in the document]
Comment (julien-c) [reply docs/hub/api.md:142]: i think it's not really used anymore (cc @lewtun @lvwerra @lhoestq maybe if they know?)

and the name is a bit confusing (if i'm a user, i'm like "are those metrics about HF?")

so i'd undocument it

[the remark this answers is not in the document]
Comment (lvwerra) [reply docs/hub/api.md:142]: no strong opinion. fine with me to remove it there.

[the remark this answers is not in the document]
Comment (lhoestq) [reply docs/hub/api.md:142]: I haven't seen it used personally

[the remark this answers is not in the document]
Comment (Wauplin) [reply docs/hub/api.md:142]: Makes sense. Let's merge this PR then :+1:

**Pull request merged**